include/cub3d.h

```diff
@@ -133,6 +133,7 @@ typedef struct s_dda
 	double	hzt_dist;
 	double	vrt_dist;
 	double	strip_height;
+	int		wall_impact;
```

Comment:
So this is the x coords relative to the wall, right?

Reply:
yes it is !